velox/common/memory/SharedArbitrator.h

```diff
@@ -553,10 +560,11 @@ class SharedArbitrator : public memory::MemoryArbitrator {
   // Lock used to protect the arbitrator internal state.
   mutable std::mutex stateLock_;
 
+  bool shutdown_{false};
```

Comment:
s/shutdown_/isShutdown_/
to indicate a state. and tsan_atomic on this?

Reply:
It is also used by global arbitration thread control and is always accessed under lock

Reply:
Yeah I understand. Just concerned that tsan test might complain.